### PR TITLE
[Bugfix] Exec Summary and Chapter 01

### PR DESCRIPTION
**Pull request by willchrist**
[BUG // CHAPTER 01 // Image stretch](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70879911)
[BUG // CHAPTER 01 // HEADING SECTION // MOBILE](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70880795)
[CHANGE REQUEST // MOBILE NAV](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70881059)
[CHANGE REQUEST // NEXT SECTION MODULE](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70880892)
[BUG // CHAPTER 01 // CARDS // MOBILE](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70880584)
[BUG // MENU LABELS AND HOVER STATE](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=70879570)
[BUG // Executive Summary // Report](https://github.com/orgs/RGADigital/projects/2/views/1?pane=issue&itemId=69805039)

### CODE REVIEW

File: src/components/atoms/button/Button.tsx [hunk 1]
```diff
@@ -18,20 +19,22 @@ export default function Button({
...
       {...buttonFramerConfig}
       className={cn(
         'dn-button',
-
-        '!h-[28px] lg:!h-[41px]',
-        'px-4 lg:!px-6',
         '!text-m-nav lg:!text-nav',
 
-        'rounded-[16px] lg:rounded-[46px]',
+        '',
```
Comment (rga-poppy) [inline src/components/atoms/button/Button.tsx:31]: the line is no longer needed @willchrist

File: src/components/atoms/icon/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      height={height}
+      viewBox="0 0 33 33"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+    >
+      <clipPath id="a">
+        <path d="m.376953.5h32v32h-32z" />
+      </clipPath>
+      <g clip-path="url(#a)">
```
Comment (rga-poppy) [inline src/components/atoms/icon/Menu.tsx:16]: replace the clip-path to clipPath to match react DOM camel case convention
https://react.dev/reference/react-dom/components#custom-html-elements

before:
```tsx
<g clip-path="url(#a)">
```
after:
```tsx
<g clipPath="url(#a)">
```

File: src/components/atoms/icon/MenuClose.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    >
+      <g clip-path="url(#clip0_2061_3968)">
+        <circle
+          cx="16"
+          cy="16"
+          r="15.3333"
+          transform="rotate(90 16 16)"
+          fill="black"
+          stroke="black"
+          stroke-width="1.33333"
```
Comment (rga-poppy) [inline src/components/atoms/icon/MenuClose.tsx:21]: stroke-width to strokeWidth

**Review by rga-poppy — COMMENTED** [review]
Hey @willchrist great work on the bugs!
I left minor comments

File: src/components/atoms/button/Button.tsx (hunk 1 above)
Comment (willchrist) [reply src/components/atoms/button/Button.tsx]: done

File: src/components/atoms/icon/Menu.tsx (hunk 2 above)
Comment (willchrist) [reply src/components/atoms/icon/Menu.tsx]: done

File: src/components/atoms/icon/MenuClose.tsx (hunk 3 above)
Comment (willchrist) [reply src/components/atoms/icon/MenuClose.tsx]: done

**Comment by rga-poppy** [comment]
Thanks for updating, lgtm!

**Review by rga-poppy — APPROVED** [review]
LGTM!

**Pull request merged**